ui/src/components/Publication/Creation/LinkedItems/index.tsx

```diff
@@ -156,7 +158,7 @@ const Links: React.FC = (): React.ReactElement => {
                     entityType="PUBLICATION"
                 />
             )}
-            {!error && !!topics.length && (
+            {!error && !!topics.length && !!isProblem && (
```

Comment:
Topics length will always be 0 for other publication types other than PROBLEM.
'&& !!isProblem' shouldn't make any difference.